types/testingutils/beacon_node_versioned.go

```diff
@@ -174,8 +242,68 @@ var TestingInvalidDutySlotV = func(version spec.DataVersion) phase0.Slot {
 	switch version {
 	case spec.DataVersionBellatrix:
 		return TestingDutySlotBellatrixInvalid
+	case spec.DataVersionCapella:
+		return TestingDutySlotCapellaInvalid
 
 	default:
 		panic("unsupported version")
 	}
 }
+
+// taken from https://github.com/attestantio/go-eth2-client/blob/1a94a0a534b9d75087403954c33f2a5819a896a9/spec/capella/beaconblock_test.go#L133
+// the block adjusted with the following changes:
+// - body.deposits proofs: 0x000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20 -> 0x000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f (they have wrong size) https://github.com/attestantio/go-eth2-client/issues/54
+// - slot: 1 -> TestingDutySlotCapella ----> to keep the consistency with TestingProposerDutyV Capella slot
```

Comment:
`TestingDutyCapella` is not 1...

Reply:
what we mean here is instead slot 1 use TestingDutySlotCapella